main.go

```diff
@@ -146,6 +145,54 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		} else if errors.Is(err, fs.ErrNotExist) {
+			srcDir, err := os.Getwd()
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusInternalServerError)
+				return
+			}
+
+			// determine package name to use for go get and go build
+			getPkg := ""
+			buildPkg := "."
+			buildDir := srcDir
+			if len(flag.Args()) > 0 {
+				pkg := flag.Args()[0]
+
+				if _, err := os.Stat(pkg); err != nil {
```

Comment:
I realized that we should also detect `pkg` starts with `./` or `.\`.

Reply:
I don't think it matters if `pkg` is relative or not, stat works just fine with absolute and relative paths. This is consistent with go build's behaviour

Reply:
I'm working on adding a check which determines if the specified package is within the current module, to avoid fetching the module again unnecessarily

Reply:
Sorry I was confused. 

Probably isn't this logic better?

```go
if strings.Contains(pkg, '@') {
    // Buiding Wasm at a temporary directory with a temporary go.mod
} else {
    // Building Wasm in the current directory
}
```

Reply:
I don't entirely trust the presence of @ to indicate a versioned package name. Consider `./m@in.go`

Reply:
I'm rather using `go list` to determine if the package is locally available or if a temporary module needs to be created to fetch it

Reply:
Hmm, but `go list` doesn't support versioned package names...

Reply:
Or, always building in the current working directory + using go-install with a temporary `GOBIN` might be an easier solution.

For example, `gomobile` analysed the current go.mod state and created a new go.mod based on it. (https://github.com/golang/mobile/blob/master/cmd/gomobile/bind.go#L296-L313) I think this is an overkill.